include/superior_mysqlpp/low_level/dbdriver.hpp

```diff
@@ -70,9 +70,6 @@ namespace SuperiorMySqlpp { namespace LowLevel
         const std::uint_fast64_t id;
         MYSQL mysql;
         Loggers::SharedPointer_t loggerPtr;
-
-        using size_t = unsigned long long;
```

Comment:
This is API breaking change, it should be mentioned in changelog at least.

Reply:
This definition is in private section.